pkgs/applications/audio/cider/default.nix

```diff
@@ -0,0 +1,33 @@
+{ appimageTools, lib, fetchurl }:
+
+appimageTools.wrapType2 rec {
+  pname = "cider";
+  version = "1.3.1308";
+  name = "${pname}-${version}";
```

Comment:
```suggestion
```
no longer required

Reply:
I originally committed this but then reverted it as it caused tests to fail.

Reply:
You mean line 15? That also accepts pname+version.

Reply:
Nope I switched that too (line 16) and tests still failed.

Reply:
How do they fail? I need some more information to better help you.

Reply:
@SuperSandro2000 I've just removed it again so you can see in the checks. The basic eval check fails.